src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I'm really never sure what's preferable honestly.
I feel like sometimes I try to call the method directly and am told to use events, and other times the opposite is true.
I don't mind either way :)